README.md

```diff
@@ -33,6 +33,9 @@ If the default SSH user is not the root user, the default user must have passwor
 |local-hooks         | |No      |Locally run hooks at different stages in the cluster setup process. See below for details|
 |on_hook_failure     |fail  |    |Behavior when hooks fail. Anything other than `fail` will `continue`|
 |install-verbosity   | |No      | Verbosity of the icp ansible installer. -v to -vvvv. See ansible documentation for verbosity information |
+|install-command     |install |No      | Installer command to run |
+|cluster-directory   |/opt/ibm/cluster |No      | Location to use for the cluster directory |
+cluster-directory
```

Comment:
I think this empty `cluster-directory` line is a typo, right?

Reply:
Correct